### PR TITLE
Feature/add adapter for geoportail lu

### PR DESCRIPTION
**Pull request by mdebo**
Hi,
I have created an adapter for geoportail.lu.
This api allows to find coordinates and address in Luxembourg.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,5 @@ rdoc/*
 .bundle
 Gemfile.lock
 api_keys.yml
+.idea/
```
Comment (TrangPham) [inline .gitignore]: This should be removed from the PR.

**Comment by TrangPham** [comment]
This looks good so far. You've added the fixtures, but your tests are not actually using them. Could you add tests which call `Geocoder.search` (i.e. one for no result, reverse result, and boulevard royal)?

**Comment by mdebo** [comment]
@TrangPham Thank you for your review. I have added some tests, and deleted .idea/* from .gitignore
Is it ok for you?

**Comment by TrangPham** [comment]
@mdebo Thanks for making those changes. :) This looks ready to merge, but could you squash your commits into one commit? This will help keep our commit log readable.

**Comment by mdebo** [comment]
@TrangPham Of course :-) Is it ok like that?

File: test/test_helper.rb [hunk 2]
```diff
@@ -283,6 +283,17 @@ def default_fixture_filename
         "#{fixture_prefix}_romsey"
       end
     end
+
```
Comment (TrangPham) [inline test/test_helper.rb]: @mdebo The tests have failed due to the merge of #970.
To fix them, add `require 'geocoder/lookups/geoportail_lu'` here.

**Comment by mdebo** [comment]
@TrangPham thank you the build is now green :-)

**Comment by alexreisner** [comment]
@mdebo thanks for your patience and good work on this. If you can just re-squash those commits into one, this is ready to merge.

**Pull request closed without merging**